src/main/java/org/fountainmc/forge/mixin/server/MixinMinecraftServer.java

```diff
@@ -66,14 +65,17 @@
     @Shadow
     public abstract int getServerPort();
 
+    private final PluginManager PLUGIN_MANAGER = new PluginManager();
```

Comment:
Names these `pluginManager` and `commandManager` since they're not public static fields.